apps/meteor/app/api/server/api.d.ts

```diff
@@ -68,7 +68,7 @@ type Options = (
 			twoFactorOptions?: ITwoFactorOptions;
 	  }
 ) & {
-	validateParams?: ValidateFunction;
+	validateParams?: ValidateFunction | { [key in Method]?: ValidateFunction };
```

Comment:
@ggazzo , the idea is to have `validateParams: { GET: xxxx, POST: xxxx }` so individual handlers can have its own validator